### ISSUE TITLE
comparison with Belgian DAISIE dataset

### ISSUE THREAD

**Issue by timadriaens** [trias-project/unified-checklist#60]
I would like to start this issue as a forms of quality control of the Belgian GRIIS checklist after some in vain attempts to discuss this at overcrowded TrIAS core group meetings and some discussions with @sanderdevisscher but we lacked to time to address this. After all this is also one of the main reasons for having published DAISIE within the AlienCSI STSM. The procedure is relatively simple I think, the big work will be in finding the appropriate source to complement GRIIS Belgium:

- select belgian species from DAISIE checklist, as `locality = Belgium`
- do an outer join with GRIIS Belgium based on common field (taxonkey) to see which species GRIIS Belgium does not have and DAISIE does and vice versa
- in depth quality control based on the fields `occurrenceStatus`, `establishmentMeans`, `eventdata` be used to explore differences with GRIIS Belgium and perhaps to improve it provided we have sources - we will also have an idea on mistakes in both registers
- `region_of_first_record` is also interesting

**Comment by peterdesmet** [comment]
There are 421 species in the DAISIE checklist that are listed as present and introduced in Belgium (i.e. [this filter](https://trias-project.github.io/unified-checklist/1_get_taxa.html#filter-on-distribution)). Of those:

1. 124 (29%) are included in the unified checklist.
2. 44 (10%) don't have a match with the GBIF backbone
3. 253 (60%) have a match with the GBIF backbone, but they are not included in the unified list

So there seems to be hardly any (124) overlap between the unified checklist and daisie. Here are the ones in daisie that do not occur in the unified: [daisie_not_in_unified.txt](https://github.com/trias-project/unified-checklist/files/4752673/daisie_not_in_unified.txt)

**Comment by qgroom** [comment]
A large proportion seem to be agricultural pest species. I guess we have overlooked these in TrIAS.

**Comment by timadriaens** [comment]
@peterdesmet could there be something wrong with the query? I notice several species that I think should be on the unified (e.g. probably all the molluscs, Anguillicoloides crassus = Anguillicola crassus, Hippolyte desmarestii = Atyaephyra desmaresti, Chrysolina americana, Magallana gigas = Crassostrea gigas and there are many many more that I recognize). 

Probably, as usual, issues with taxon names, synonyms etc.? I thought these would be easy to filter out with daisie and unified both mapped onto gbif?

**Comment by timadriaens** [comment]
Sporobolus townsendii = Spartina townsendii
Anodonta woodiana = Sinanodonta woodiana

**Comment by timadriaens** [comment]
a lot of the species seem to be marine, molluscs or macroinverts

**Comment by timadriaens** [comment]
and the inevitable "Ve velutina nigrithorax"

**Comment by peterdesmet** [comment]
Ve velutina was an error that affected many taxa: https://github.com/trias-project/daisie-checklist/issues/37 I will run it again.

**Comment by peterdesmet** [comment]
New results:

- 24 cannot be matched to GBIF: expected, some DAISIE scientific names are really crappy: "Macrosiphum Macrosiphum albifrons Essig, 1911 aphidoidea". I wonder if we should invest time in cleaning these??
- 129 (5 more) now match with the unified
- 268 are not in unified: [daisie_not_in_unified.txt](https://github.com/trias-project/unified-checklist/files/4786494/daisie_not_in_unified.txt). This included "Vespa velutina nigrithorax du Buysson, 1905" which is a subspecies, while the unified has the species.

**Comment by sannegovaert** [comment]
@peterdesmet @timadriaens Has anything been done with this data gap?

**Comment by peterdesmet** [comment]
No. Since then the waarnemingen.be checklist has been added to the unified, which likely filled many gaps.
